src/main/java/net/snowflake/client/jdbc/SnowflakeTimestampWithSessionTimezone.java

```diff
@@ -6,20 +6,32 @@
 
 import java.sql.Timestamp;
 import java.time.LocalDateTime;
+import java.time.ZoneId;
 import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
+import java.util.TimeZone;
 
-/** Timestamp with toString in UTC timezone. */
-public class SnowflakeTimestampNTZAsUTC extends Timestamp {
+/**
+ * Timestamp with toString() overridden to display timestamp in session timezone. The default
+ * timezone is UTC if no timezone is specified.
+ */
+public class SnowflakeTimestampWithSessionTimezone extends Timestamp {
```

Comment:
I feel like from the class itself, it's not very clear why it needs to be called as SnowflakeTimestampWithSessionTimezone. Do you think SnowflakeTimestampWithTimezone is a not bad alternative name for the class?

Reply:
This is true, I can take out the Session name since it relates more to just timezone rather than session timezone. I will change these 3 class names to take out the "Session".